module-2/backend/Liam_barstad/week_one.txt

```diff
@@ -0,0 +1,92 @@
+List the five common HTTP verbs and what the purpose is of each verb.
+  - get: to fetch views content
+  - post: to upload content and/or add to database
+  - delete: to delete content and/or remove from database
+  - put: ??
+  - update: to rewrite content and/or replace elements in the database
```

Comment:
Update isn't an HTTP verb. PATCH is the last one we were looking for, which (along with PUT) also updates a resource.